### PR TITLE
Add TESTING.md describing testing details

### PR DESCRIPTION
**Pull request by kkaarreell**
Signed-off-by: Karel Srot <ksrot@redhat.com>

### CODE REVIEW

File: TESTING.md [hunk 1]
```diff
@@ -0,0 +1,141 @@
+# Keylime testing
+
+## Pull-request testing
+
+Individual changes to keylime are introduced using the pull-request workflow and for each pull-request update a set of test is executed.
+
+### Keylime testsuite
```
Comment (THS-on) [inline TESTING.md:7]: Do you want to add also a reference to the `.ci/run_local.sh` script?

Comment (kkaarreell) [reply TESTING.md:7]: @THS-on Would you suggest some wording? I am not familiar with it so I do not want to add something misleading.

Comment (THS-on) [reply TESTING.md:7]: The Keylime test suite can be run locally by executing `.ci/run_local.sh` (requires Docker).

Comment (kkaarreell) [reply TESTING.md:7]: I have added Local execution section at the very end of the doc, please review.

**Review by THS-on — APPROVED** [review]
LGTM.

**Pull request merged**